crates/primitives/convert/src/felt.rs

```diff
@@ -24,6 +25,23 @@ pub fn felt_to_u128(felt: &Felt) -> Result<u128, FeltToU128Error> {
     }
 }
 
+#[derive(Debug, thiserror::Error)]
+#[error("Felt is too big to convert to H160.")]
+pub struct FeltToH160Error;
+
+pub fn felt_to_h160(felt: &Felt) -> Result<H160, FeltToH160Error> {
+    let bytes = felt.to_bytes_be();
+
+    // check if the 12 first bytes are null
+    if !bytes.iter().take(12).all(|&b| b == 0) {
```

Comment:
Why not a simple comparison of the type `felt > 2^160 - 1` (pseudocode) with a static `from_hex_unchecked` ?